src/features/identityCheck/pages/identification/IdentityCheckEduConnectForm.tsx

```diff
@@ -50,11 +51,26 @@ export const IdentityCheckEduConnectForm = () => {
     const isError = event.url.includes('educonnect/erreur')
     if (isError) {
       if (event.url.includes('UserAgeNotValid18YearsOld')) {
-        setError(new EduConnectError(EduConnectErrors['not-eligible'], 'UserAgeNotValid18YearsOld'))
+        setError(
+          new EduConnectError(
+            EduConnectErrors['not-eligible'],
+            NotEligibleEduConnectErrorMessageEnum.UserAgeNotValid18YearsOld
+          )
+        )
       } else if (event.url.includes('UserAgeNotValid')) {
-        setError(new EduConnectError(EduConnectErrors['not-eligible'], 'UserAgeNotValid'))
+        setError(
+          new EduConnectError(
+            EduConnectErrors['not-eligible'],
+            NotEligibleEduConnectErrorMessageEnum.UserAgeNotValid
+          )
+        )
       } else if (event.url.includes('UserTypeNotStudent')) {
-        setError(new EduConnectError(EduConnectErrors['not-eligible'], 'UserTypeNotStudent'))
+        setError(
+          new EduConnectError(
+            EduConnectErrors['not-eligible'],
```

Comment:
Pourquoi on a besoin de cette clé "not-eligible" ? Je vois pas où on en utilise une autre

Reply:
Je devrais pouvoir faire sans, je regarde !